dataset_generation/hi_wikisplit/Scraper.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import requests
+import os
+import sys
+import shutil
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+class Scraper:
+    """
+    Uses the wikimedia api to get the revision histories of the titles 
+    provided in the title dump file.
+    Format varies between wikipedia languages
+    """
+    def __init__(self, titles_file, url = "https://hi.wikipedia.org/w/api.php",
+                 titles_per_chunk = 1000, output_dir = './raw_data/',
+                 max_rev_ids = 500, start_from=0): 
+        
+        """
+        titles_file: title dump file
+        titles_per_chunk: chunk size, data gets written in chunks
+        output_dir: data chunks will be written to this directory
+        max_rev_ids: gets at most the specified number of revisions for a page. 
+                    500 is the upper limit
+        url: url to scrape from
```

Comment:
can we organize the comments in the order of the function arguments i.e. url after titles_file

Reply:
also, can we move url also as a required argument since this url is specific to hindi?

Reply:
yes, sorry about that!